scripts/Utils.py

```
@@ -0,0 +1,127 @@
+def GetSystemEnvironmentVariable(name):
+    key = winreg.CreateKey(winreg.HKEY_LOCAL_MACHINE, r"System\CurrentControlSet\Control\Session Manager\Environment")
+    try:
+        return winreg.QueryValueEx(key, name)[0]
+    except:
+        return None
+
+def GetUserEnvironmentVariable(name):
+    key = winreg.CreateKey(winreg.HKEY_CURRENT_USER, r"Environment")
+    try:
+        return winreg.QueryValueEx(key, name)[0]
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)